lib/openstax/path_prefixer/action_controller_base_extensions.rb

```diff
@@ -18,7 +18,7 @@ def _compute_redirect_to_location(request, options) #:nodoc:
         already_has_prefix = options.match(/^#{request.script_name}/)
 
         unless already_has_prefix
-          options_uri = URI(options)
+          options_uri = Addressable::URI.parse(options)
```

Comment:
`require 'addressable/uri'` at the top (gemspec does not load dependencies automatically)

Reply:
Done